### ISSUE TITLE
Packer crash  for version 0.9.0 (Windows 2012 R2 Template Build) 

### ISSUE THREAD

**Issue by les-tallman** [mitchellh/packer#3286]
Describe the problem and include the following information: Attempting to build a Windows 2012 R2 Template and Packer crashed.
- Packer Version 0.9.0
- Host platform CentoS 7 with VMware Workstation 12

[crash_02_29.log.zip](https://github.com/mitchellh/packer/files/151907/crash_02_29.log.zip)
[template_02_29.json.zip](https://github.com/mitchellh/packer/files/151924/template_02_29.json.zip)
[vmware-iso.zip](https://github.com/mitchellh/packer/files/151925/vmware-iso.zip)
vmware-iso.zip contains the autounattend & powershell scripts as requested.

```
2016/02/29 17:42:16 packer: panic: runtime error: invalid memory address or nil pointer dereference
2016/02/29 17:42:16 packer: [signal 0xb code=0x1 addr=0x58 pc=0xe86b86]
2016/02/29 17:42:16 packer: 
2016/02/29 17:42:16 packer: goroutine 203 [running]:
2016/02/29 17:42:16 packer: panic(0x11790e0, 0xc82000e0f0)
2016/02/29 17:42:16 packer: 	/usr/local/go/src/runtime/panic.go:464 +0x3e6
2016/02/29 17:42:16 packer: github.com/mitchellh/packer/vendor/github.com/packer-community/winrmcp/winrmcp.cleanupContent(0xc82023e080, 0xc8202c8080, 0x3a, 0x0, 0x0)
2016/02/29 17:42:16 packer: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/vendor/github.com/packer-community/winrmcp/winrmcp/cp.go:169 +0x186
2016/02/29 17:42:16 packer: github.com/mitchellh/packer/vendor/github.com/packer-community/winrmcp/winrmcp.doCopy(0xc82023e080, 0xc8204902d0, 0x7f0470c02000, 0xc82006e1a0, 0xc820490280, 0x4a, 0x0, 0x0)
2016/02/29 17:42:16 packer: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/vendor/github.com/packer-community/winrmcp/winrmcp/cp.go:42 +0xc97
2016/02/29 17:42:16 packer: github.com/mitchellh/packer/vendor/github.com/packer-community/winrmcp/winrmcp.(*Winrmcp).Write(0xc820240340, 0xc820490280, 0x4a, 0x7f0470c02000, 0xc82006e1a0, 0x0, 0x0)
2016/02/29 17:42:16 packer: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/vendor/github.com/packer-community/winrmcp/winrmcp/winrmcp.go:78 +0x8d
2016/02/29 17:42:16 packer: github.com/mitchellh/packer/communicator/winrm.(*Communicator).Upload(0xc8203a6140, 0xc820490280, 0x4a, 0x7f0470c02000, 0xc82006e1a0, 0x0, 0x0, 0x0)
2016/02/29 17:42:16 packer: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/communicator/winrm/communicator.go:126 +0x1a8
2016/02/29 17:42:16 packer: github.com/mitchellh/packer/packer/rpc.(*CommunicatorServer).Upload(0xc82039ad20, 0xc8203a6320, 0xc820240220, 0x0, 0x0)
2016/02/29 17:42:16 packer: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/packer/rpc/communicator.go:270 +0x15e
2016/02/29 17:42:16 packer: reflect.Value.call(0x10c1820, 0x12799a8, 0x13, 0x13c8af8, 0x4, 0xc82038fed8, 0x3, 0x3, 0x0, 0x0, ...)
2016/02/29 17:42:16 packer: 	/usr/local/go/src/reflect/value.go:435 +0x120d
2016/02/29 17:42:16 packer: reflect.Value.Call(0x10c1820, 0x12799a8, 0x13, 0xc82038fed8, 0x3, 0x3, 0x0, 0x0, 0x0)
2016/02/29 17:42:16 packer: 	/usr/local/go/src/reflect/value.go:303 +0xb1
2016/02/29 17:42:16 packer: net/rpc.(*service).call(0xc8203a56c0, 0xc8203a5680, 0xc8202400f8, 0xc8202d6680, 0xc8203a6220, 0xec8720, 0xc8203a6320, 0x16, 0xed82c0, 0xc820240220, ...)
2016/02/29 17:42:16 packer: 	/usr/local/go/src/net/rpc/server.go:383 +0x1c2
2016/02/29 17:42:16 packer: created by net/rpc.(*Server).ServeCodec
2016/02/29 17:42:16 packer: 	/usr/local/go/src/net/rpc/server.go:477 +0x49d
2016/02/29 17:42:16 /opt/packer/bin/packer: plugin process exited
```

**Comment by cbednarski** [comment]
Thanks for the report and crash logs.

For later reference, this crash occurs while uploading a script over WinRM, immediately after the VM is available.

I think https://github.com/mitchellh/packer/issues/3233 is the same issue so I'm going to close that one since this one includes a repro.

**Comment by mwhooker** [comment]
fixed via #4424

It won't fix the underlying issue, but it will stop the panic.

The underlying issue will be reported, and if it's not something you're able to solve, please open another issue.
